modules/users/client/controllers/settings/change-password.client.controller.js

```diff
@@ -33,11 +33,13 @@
       // If successful show success message and clear form
       $scope.$broadcast('show-errors-reset', 'vm.passwordForm');
       vm.success = true;
```

Comment:
Is it possible to modify this test (and others like it) to check that the toast was opened?